examples/local/501_teardown.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# Copyright 2019 The Vitess Authors.
```

Comment:
Should be 2024
```suggestion
# Copyright 2024 The Vitess Authors.
```

Reply:
We're only noticing this because it's not a file move/rename. I thought that we did that on the call earlier today. Maybe we just didn't push it yet?